formatter.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
```

Comment:
The [gruntwork-cli](https://github.com/gruntwork-io/gruntwork-cli) library already has logrus built in and configured. Would it make more sense to use that? Ideally, we'd move all our CLI tools to use this library under the hood. Perhaps using it just for logging here is an incremental first step?

Reply:
I looked at Terragrunt and noticed somebody had contributed a PR with `logrus` about a week ago so I thought that might have been the way forward, but I'm happy to use `gruntwork-cli`. I noticed this repo is using a much older version of `https://github.com/urfave/cli`, so I was a bit worried about compatibility issues. Should I look into swapping it over?

Reply:
Yeah I just realized I could only use `gruntwork-cli` for logging as a first step like you mentioned then move over to the other collections at a later stage. I'll port this over.

Reply:
Roger!

Reply:
Ok, refactored to use `gruntwork-cli` instead for logging.